{
    "repository": "bonitasoft/bonita-engine",
    "clone_url": "https://github.com/bonitasoft/bonita-engine.git",
    "timestamp": "2023-05-28T17:53:35.001707Z",
    "clone_success": true,
    "number of actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[Build/Build] \ud83d\ude80  Start image=crawlergpt:latest\n[Build/Build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[Build/Build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Build/Build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Build/Build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/bc275e72-fd77-11ed-a890-af2cc187fc11/bonitasoft-bonita-engine] user=0 workdir=\n[Build/Build]   \u2601  git clone 'https://github.com/actions/setup-java' # ref=v1\n[Build/Build] Non-terminating error while running 'git clone': some refs were not updated\n[Build/Build]   \u2601  git clone 'https://github.com/actions/cache' # ref=v1\n[Build/Build]   \u2601  git clone 'https://github.com/actions/cache' # ref=v1\n[Build/Build] \u2b50 Run Main actions/checkout@v2\n[Build/Build]   \u2705  Success - Main actions/checkout@v2\n[Build/Build] Error response from daemon: No such container: 464208ad3b6447364c051be67b56a4b459bd6a810f7e16be88101ca858a8953c\n[Build/Build] \u2b50 Run Main Set up JDK 11\n[Build/Build]   \ud83d\udc33  docker cp src=/home/andre-silva/.cache/act/actions-setup-java@v1/ dst=/var/run/act/actions/actions-setup-java@v1/\n[Build/Build]   \u274c  Failure - Main Set up JDK 11\n[Build/Build] Error response from daemon: No such container: 464208ad3b6447364c051be67b56a4b459bd6a810f7e16be88101ca858a8953c\n[Build/Build] \u2b50 Run Post Set up JDK 11\n[Build/Build]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-java@v1/dist/cleanup/index.js] user= workdir=\n[Build/Build]   \u274c  Failure - Post Set up JDK 11\n[Build/Build] failed to remove container: Error response from daemon: No such container: 464208ad3b6447364c051be67b56a4b459bd6a810f7e16be88101ca858a8953c\n[Build/Build] \ud83c\udfc1  Job failed\n",
    "actions_stderr": "Error: Error occurred running finally: Error occurred running finally: Error occurred running finally: Error occurred running finally: Error response from daemon: No such container: 464208ad3b6447364c051be67b56a4b459bd6a810f7e16be88101ca858a8953c (original error: <nil>) (original error: <nil>) (original error: <nil>) (original error: <nil>)\n"
}